### DISCUSSION TITLE
paddlepaddle2.6.2 paddleocr要安装哪个版本的cuda？nvidia 驱动更新后是12.7

### DISCUSSION THREAD

**Discussion by nissansz** [PaddlePaddle/PaddleOCR#14100] [Q&A]
paddlepaddle2.6.2 paddleocr要安装哪个版本的cuda？nvidia 驱动更新后是12.7

**Comment by GodCzzZ** [comment]
2.6 paddlepaddle 适配的nvidia CUDA 版本 最高是12.0，但是 2.6.2 不适配12.0吧

**Comment by nissansz** [comment]
你是什么版本？什么系统？能训练吗？









--------------------------------------------------------------------------------





------------------ 原始邮件 ------------------




 
 发件人: GodCzzZ  ***@***.***>
 

 
 发送时间: 2024-10-26 14:09:52
 

 
 收件人:PaddlePaddle/PaddleOCR ***@***.***> 

 
 抄送:nissanjp ***@***.***>,Author ***@***.***> 

 
 主题: Re: [PaddlePaddle/PaddleOCR] paddlepaddle2.6.2 paddleocr要安装哪个版本的cuda？nvidia 驱动更新后是12.7 (Discussion #14100)
 





 2.6 paddlepaddle 适配的nvidia CUDA 版本 最高是12.0，但是 2.6.2 不适配12.0吧

**Reply by GodCzzZ** [reply]
你的12.7要卸载重装，飞桨的官网有具体要求的，paddlepaddle版本，cudn版本，cudnn版本，就这三个需要对应

**Comment by nissansz** [comment]
我是下载nvidia驱动管理软件，它自动检测最新驱动，只能看到12.7









--------------------------------------------------------------------------------





------------------ 原始邮件 ------------------




 
 发件人: GodCzzZ  ***@***.***>
 

 
 发送时间: 2024-10-26 14:15:45
 

 
 收件人:PaddlePaddle/PaddleOCR ***@***.***> 

 
 抄送:nissanjp ***@***.***>,Author ***@***.***> 

 
 主题: Re: [PaddlePaddle/PaddleOCR] paddlepaddle2.6.2 paddleocr要安装哪个版本的cuda？nvidia 驱动更新后是12.7 (Discussion #14100)
 





 你的12.7要卸载重装，飞桨的官网有具体要求的，paddlepaddle版本，cudn版本，cudnn版本，就这三个需要对应

**Reply by GodCzzZ** [reply]
nvidia官网有历史版本，百度上有很多安装paddleocr教程的，感觉你是个萌新

**Reply by nissansz** [reply]
https://www.paddlepaddle.org.cn/documentation/docs/zh/install/pip/windows-pip.html

驱动时安装最新的，cuda, cudnn才需要指定以下版本吧？

如果您的计算机有 NVIDIA® GPU，请确保满足以下条件并且安装 GPU 版 PaddlePaddle

CUDA 工具包 11.8 配合 cuDNN v8.6.0，如需使用 PaddleTensorRT 推理，需配合 TensorRT8.5.1.7

CUDA 工具包 12.3 配合 cuDNN v9.0.0, 如需使用 PaddleTensorRT 推理，需配合 TensorRT8.6.1.6

GPU 运算能力超过 6.0 的硬件设备

**Comment by nissansz** [comment]
你是什么版本？能训练吗？











--------------------------------------------------------------------------------





------------------ 原始邮件 ------------------




 
 发件人: GodCzzZ  ***@***.***>
 

 
 发送时间: 2024-10-26 14:23:58
 

 
 收件人:PaddlePaddle/PaddleOCR ***@***.***> 

 
 抄送:nissanjp ***@***.***>,Author ***@***.***> 

 
 主题: Re: [PaddlePaddle/PaddleOCR] paddlepaddle2.6.2 paddleocr要安装哪个版本的cuda？nvidia 驱动更新后是12.7 (Discussion #14100)
 





 nvidia官网有历史版本，百度上有很多安装paddleocr教程的，感觉你是个萌新

**Comment by GreatV** [comment]
推荐使用paddlex，低代码一站式训练。别折腾这环境配置了。

**Comment by nissansz** [comment]
paddlex不能训练最新的配置文件吧









--------------------------------------------------------------------------------





------------------ 原始邮件 ------------------




 
 发件人: Wang Xin  ***@***.***>
 

 
 发送时间: 2024-10-26 14:35:34
 

 
 收件人:PaddlePaddle/PaddleOCR ***@***.***> 

 
 抄送:nissanjp ***@***.***>,Author ***@***.***> 

 
 主题: Re: [PaddlePaddle/PaddleOCR] paddlepaddle2.6.2 paddleocr要安装哪个版本的cuda？nvidia 驱动更新后是12.7 (Discussion #14100)
 





 推荐使用paddlex，低代码一站式训练。别折腾这环境配置了。

**Reply by GreatV** [reply]
你问问paddlex仓库吧，应该问题不大

**Reply by nissansz** [reply]
paddlex 是要用ai studio训练？还是本地训练？
